doc/content/integrations/mqtt/Connecting to Node-RED/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Connecting to Node-RED"
+description: ""
+---
+
+[Node-RED](https://nodered.org/) is free, JavaScript-based development tool for visual programming, developed to ease the process of wiring together hardware devices, APIs and online services.
+
+The MQTT server that is exposed by the Application Server can be connected to Node-RED. This kind of integration allows setting up a Node-RED flow that listens to events and messages sent by the end devices. Also, it is possible to send messages to the end devices.
+
+This guide contains the instructions for setting up these flows, but first make sure to properly [install the Node-RED](https://nodered.org/docs/getting-started/local).
+
+## Setup
+
+First, you need to run the Node-RED by using the following command in terminal:
+```bash
+$ node-red 
+```
+```bash
+# Tip: Node-RED runs locally on port 1880 by default, so make sure to allow the service to run on this port with:
+
+$ sudo ufw allow 1880
+```
+
+>Note: you have to [secure Node-RED](https://nodered.org/docs/user-guide/runtime/securing-node-red) if you want to connect it to MQTT Server over TLS, since its editor is not secured by the default.
+
+If Node-RED is sucessfully launched, when navigating to `https://localhost:1880` in your web browser you should see something like:
+
+{{< figure src="nodered_dashboard.png" alt="Node-RED dashboard" >}}
+
+On the left side, you can see various types of nodes (they can also be found in the [Node-RED library](https://flows.nodered.org/)) that can be used in order to build flows, although in this guide only a few simple inbuilt nodes are used. 
+
+The Console provides the information needed for completing this integration, under **MQTT** submenu from **Integrations** menu within desired application.
+
+{{< figure src="console_info.png" alt="MQTT Server connection information" >}}
+
+In this example, since TTS is running locally, the inbuilt MQTT Server is deployed on port 1883 for insecure connection and on port 8883 for the TLS-secured one.
+
+## Receive events and messages
+
+This section follows the process of setting up a flow which subscribes and listens to the events and messages that are being published by the MQTT Server. 
+
+First, find the `mqtt in` node in the library section and place it on the flow dashboard. This node connects to the MQTT broker and subscribes to messages from the specified topic. Double-click on the node to configure its properties.
```

Comment:
I think we should also use `**Strong**` when referring to nodes, similar to how we style text that refers to buttons.